### DISCUSSION TITLE
Trailing slashes in the endpoint URLs

### DISCUSSION THREAD

**Discussion by mderriey** [svix/svix-webhooks#1252] [Q&A] [answered]
Hey there

We got caught twice by the trailing slashes in the URLs of the endpoints while playing around with Postman and hand-written code.

We're aware that the OpenAPI document is there to avoid potential mistakes, and that you provide SDKs to integrate with the API, but I feel like it's an uncommon pattern.

Some questions:

- Out of curiosity, why?
- Any chance the API could support not having the trailing slashes while being backwards-compatible?

As an FYI, we used Microsoft's Kiota to generate a .NET API client, and during generation it removed the trailing slashes. It's definitely a bug on their side (which we filed over at <https://github.com/microsoft/kiota/issues/4291>), but it may show that it's not "standard".

**Comment by tasn** [comment]
Mind opening a ticket about supporting both? It was a mistake that we very much regret, but I guess supporting both wouldn't be too much of a hassle (or at least we can check whether it would be). Thanks!

**Reply by mderriey** [reply, marked as answer]
Issue opened in <https://github.com/svix/svix-webhooks/issues/1254>, thanks.

**Reply by tasn** [reply]
Thanks!
